todo_app/templates/index.html

```diff
@@ -11,8 +11,15 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items%}
+          <li>{{item.id}}. {{item.title}}: {{item.status}}</li>
+        {% endfor %}
       </ul>
     </div>
+    <form action="/add" method="post">
+      <label for="title">Film name:</label><br>
+      <input type="text" id="title" name="title"><br>
```

Comment:
Very minor, but annotating this as `required` would provide some simple client-side validation

Reply:
I've started module 2 as I was going on holiday before this would be merged in so I've added these changes into module 2's PR